packages/suite/src/components/suite/layouts/WelcomeLayout/WelcomeSidebar.tsx

```diff
@@ -0,0 +1,155 @@
+import { Column, IconButton, variables } from '@trezor/components';
+import { setDiscreetMode } from 'src/actions/settings/walletSettingsActions';
+import { goto } from 'src/actions/suite/routerActions';
+import { useDispatch, useSelector, useTranslation } from 'src/hooks/suite';
+import { selectIsDiscreteModeActive } from 'src/reducers/wallet/settingsReducer';
+import { TrafficLightOffset } from '../../TrafficLightOffset';
+import styled from 'styled-components';
+import { borders, spacingsPx } from '@trezor/theme';
+import { Translation } from '../../Translation';
+import { useCallback, useEffect, useState } from 'react';
+import { SettingsAnchor } from 'src/constants/suite/anchors';
+import { selectTorState } from 'src/reducers/suite/suiteReducer';
+import { useLocation } from 'react-router-dom';
+import { ButtonVariant } from '@trezor/components/src/components/buttons/buttonStyleUtils';
```

Comment:
Could you use just simple package here? 
`import ... from '@trezor/components'`